### PR TITLE
docs: modify the use of slot

### PR DESCRIPTION
**Pull request by BeADre**
- [ ] docs
 - tooltip、popover：modify the use of slot

### CODE REVIEW

**Comment by BeADre** [comment]
@JeremyWuuuuu  Is there any problem with this PR?

**Comment by JeremyWuuuuu** [comment]
Sorry, haven't got the time to review.

**Comment by BeADre** [comment]
0.0   ok。。。。。

File: website/docs/en-US/popover.md [hunk 1]
```diff
@@ -34,20 +36,23 @@ Similar to Tooltip, Popover is also built with `Vue-popper`. So for some duplica
...
 
   <el-popover
     placement="bottom"
     title="Title"
-    width="200"
+    :width="200"
     trigger="manual"
     content="this is content, this is content, this is content"
-    v-model:visible="visible">
+    :visible="visible"
```
Comment (JeremyWuuuuu) [inline website/docs/en-US/popover.md:54]: We need to use `v-model:visible` simply binding the value won't get updated after `@click` event occurs

**Review by jw-foss — COMMENTED** [review]
You need to revert `:visible` back to `v-model:visible`

**Comment by BeADre** [comment]
done

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
There are conflicts you need to resolve it before merging

**Comment by BeADre** [comment]
resolved

**Pull request merged**